### ISSUE TITLE
Option to merge all scenes/layers into a single scene/layer

### ISSUE THREAD

**Issue by niklr** [Zarbuz/VoxMerger#7]
Thank you for creating this awesome tool!

Is there an option to merge all scenes/layers into a single scene/layer? If not, any hint/recommendation would help to accomplish this programmatically.

**Comment by Zarbuz** [comment]
I can add this option if you want

**Comment by Zarbuz** [comment]
Do we talk about those layers right ? 
![image](https://user-images.githubusercontent.com/4965714/148124138-1b4b3f2c-f162-4093-bc94-05067afc10f3.png)

**Comment by niklr** [comment]
No, in MagicaVoxel it is called "Outline". They can be shown or hidden like layers e.g. in Photoshop.

![grafik](https://user-images.githubusercontent.com/4712166/148212905-486ddcbe-7ff8-4966-92b4-6a66cb3153a9.png)

**Comment by Zarbuz** [comment]
Ah so it's called regions, not layers ! 
There is another ticket about this, pretty hard task
